### DISCUSSION TITLE
Improve spatial queries to allow drawing polygons

### DISCUSSION THREAD

**Discussion by grantfitzsimmons** [specify/specify7#5812] [Feature Requests]
Currently you can only select two points. This should be expanded to allow searching within a polygon.

**Comment by maxpatiiuk** [comment]
See https://developers.arcgis.com/javascript/latest/query-a-feature-layer-spatial/ for an example implementation

**Comment by danb213** [comment]
We have had requests from our new Herbaria users at CSIRO for this feature
